relay-server/src/actors/processor.rs

```diff
@@ -1712,6 +1716,14 @@ impl EnvelopeProcessorService {
                 // hint to normalization that we're dealing with a transaction now.
                 self.event_from_json_payload(item, Some(EventType::Transaction))?
             })
+        } else if let Some(item) = user_report_v2_item {
+            relay_log::trace!("processing user_report_v2");
+            let project_state = &state.project_state;
+            let user_report_v2_ingest = project_state.has_feature(Feature::UserReportV2Ingest);
+            if !user_report_v2_ingest {
+                return Err(ProcessingError::NoEventPayload);
+            }
+            self.event_from_json_payload(item, Some(EventType::UserReportV2))?
```

Comment:
@iker-barriocanal we split event-like items into a separate envelope, so that should be fine.

Raising an error here will log an `invalid` outcome, but only if `outcome_category()` returns `Some` (see discussion below).